### PR TITLE
docs: update en-US.js

### PR DESCRIPTION
**Pull request by scouttyg**
A few changes to the homepage that make the wording flow a bit better

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: site/theme/en-US.js [hunk 1]
```diff
@@ -34,19 +34,19 @@ module.exports = {
...
-    'app.home.natural': 'Natural',
+      'This is Ant Design\'s internal standard for evaluating design quality. Based on the assumption that "everyone is pursuing happiness at work", we have added the two values of "Meaningfulness" and "Growth" on the basis of "Certainty" and "Naturalness" to guide each designer towards better judgment and decision-making.',
+    'app.home.certainty': 'Certainty',
+    'app.home.meaningful': 'Meaningfulness',
+    'app.home.growth': 'Growth',
+    'app.home.natural': 'Naturalness',
     'app.home.design-guide': 'Guidelines',
     'app.home.components': 'Components',
-    'app.home.detail': 'Detail',
+    'app.home.detail': 'More details',
```
Comment (fireairforce) [inline site/theme/en-US.js:49]: Why not `Details`？

Comment (scouttyg) [reply site/theme/en-US.js:49]: From what I've seen, usually the link to "view more information" is usually couched with the word "more" in some way -- e.g. wording like:

- "Read more"
- "More info[rmation]"
- "More details"
- "...More"

If you search google for "details" button, this somewhat seems to show that as well:

Screen Shot 2020-07-16 at 11 54 39 AM

It's not 100% always the case of course, as people also use things like:

- Continue reading
- View Details

etc, but I haven't commonly seen just a "details" link by itself -- just my two cents though!

**Review by 07akioni — APPROVED** [review]

File: site/theme/en-US.js (hunk 1 above)
Comment (fireairforce) [reply site/theme/en-US.js:49]: ok~

**Review by fireairforce — APPROVED** [review]
LGTM

**Review by xrkffgg — APPROVED** [review]

**Pull request merged**